jest.config.js

```diff
@@ -14,7 +14,7 @@ module.exports = {
         "json",
         "node"
     ],
-    "coverageReporters": ["html", "json", ["lcov", {"projectRoot": "../../"}], "text"],
+    "coverageReporters": ["html", "json", ["lcov", {"projectRoot": __dirname}], "text"],
```

Comment:
@nschonni it was this line.
The projectRoot was not correctly set.

Reply:
interesting, I think I ran into something similar for the ESLint stuff at one point too